osmoutils/accum/accum.go

```diff
@@ -77,10 +77,27 @@ func (accum *AccumulatorObject) UpdateAccumulator(amt sdk.DecCoins) {
 // The position is initialized with empty unclaimed rewards
 // If there is an existing position for the given address, it is overwritten.
 func (accum AccumulatorObject) NewPosition(name string, numShareUnits sdk.Dec, options *Options) error {
+	return accum.newPosition(name, numShareUnits, accum.value, options)
+}
+
+// NewPosition creates a new position for the given name, with the given number of share units.
+// The name can be an owner's address, or any other unique identifier for a position.
+// It sets the position's accumulator to the given value of customAccumulatorValue.
+// All custom accumulator values must be non-negative.
+// The position is initialized with empty unclaimed rewards
+// If there is an existing position for the given address, it is overwritten.
+func (accum AccumulatorObject) NewPositionCustomAcc(name string, numShareUnits sdk.Dec, customAccumulatorValue sdk.DecCoins, options *Options) error {
```

Comment:
I've considered this but tbh I prefer the original `NewPositionCustomAcc` better because it is shorter. To me, there is no semantical difference between `Custom` and `Customized`.

If you don't mind, I will keep the old name for shortness but lmk if any concerns